examples/ts-react-chat/src/routes/index.tsx

```
@@ -243,6 +272,104 @@ function ChatPage() {
+          metadata: { mediaType: img.mimeType, mimeType: img.mimeType },
+        })
+      }
+
+      // Send with custom message ID
+      sendMessage({
+        content: contentParts,
+        id: generateMessageId(),
+      })
+
+      // Clean up image previews
+      attachedImages.forEach((img) => URL.revokeObjectURL(img.preview))
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)